### ISSUE TITLE
go build reports error on mac

### ISSUE THREAD

**Issue by leyaocty** [krux02/assimp#4]
Hi, I'm using your package for model loading and it's pretty well on linux. But go reports error while building on mac as follow:

# github.com/krux02/assimp
In file included from ../../go/pkg/mod/github.com/krux02/assimp@v0.0.0-20181005083220-f01e82f24379/Anim.go:4:
In file included from /usr/local/include/assimp/anim.h:55:
/usr/local/include/assimp/quaternion.h:133:5: error: unknown type name 'ai_real'
    ai_real w, x, y, z;
    ^
1 error generated.

Would you please advise how to fix this issue on mac, thank you!

**Comment by krux02** [comment]
Well, nice to see that after 8 years somebody is using my wrapper. Unfortunately I don't know this problem, all I can do is I can try to do a far diagnosis. I wrote this wrapper on Linux with the intention to be usable on other platforms as well, but never tested it on any other platform at all.

Unfortunately I did not fixate assimp on a specific version. I just wrote a wrapper with the hopes that assimp is already important enough so it doesn't break compatibility on future changes. 

The error message you get is linked to a local file `/usr/local/include/assimp/quaternion.h:133:5` on you computer that I do not have access to, nor do I know the example that you try to compile, so I my capabilities to remote debug this problem are a bit limited.

**Comment by leyaocty** [comment]
I believe your wrap will be involved by more projects since many developers are trying to use go to drive GL due to the big advantage of concurrent spec.

Actually I passed the go compile on previous MAC OS version bug failed on last version. I checked on internet and is seems to be the compile issue while parsing c/c++ under different version. I will check it under other compilers for this. 

Thank you for you prompt response!
